__tests__/app.test.js

```diff
@@ -126,5 +126,48 @@ describe('GET /api/articles/:article_id', () => {
         .then(({ body: { msg } }) => {
             expect(msg).toBe('Invalid article_id');
         });
+  });
 });
-});
\ No newline at end of file
+
+describe('GET /api/articles/:article_id/comments', () => {
+  test('200: Responds - each comment should have the correct structure', () => {
+    const article_id = 1; 
+    return request(app)
+      .get(`/api/articles/${article_id}/comments`)
+      .expect(200)
+      .then(({ body: { comments } }) => {
+        comments.forEach((comment) => {
+          expect(comment).toEqual(
+            expect.objectContaining({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+              article_id: article_id, 
+            })
+          );
+        });
+      });
+  });
+  test('200: Responds with an array of comments for article_id + msg if there no comments', () => {
```

Comment:
You mention a `msg` in the description but don't check one in your test. I wouldn't recommend sending a message for this one.